runtime/web_worker.rs

```diff
@@ -582,6 +627,20 @@ impl WebWorker {
   ) -> Result<(), AnyError> {
     poll_fn(|cx| self.poll_event_loop(cx, wait_for_inspector)).await
   }
+
+  // Starts polling for messages from worker host from JavaScript.
+  fn start_polling_for_messages(&mut self) {
+    let poll_for_messages_fn = self.poll_for_messages_fn.take().unwrap();
+    let context_global = self.js_runtime.global_context();
+    let scope = &mut self.js_runtime.handle_scope();
+    let poll_for_messages =
+      v8::Local::<v8::Value>::new(scope, poll_for_messages_fn);
+    let fn_ = v8::Local::<v8::Function>::try_from(poll_for_messages).unwrap();
+    let context = context_global.open(scope);
+    let global = context.global(scope);
+    let global_value = v8::Local::<v8::Value>::from(global);
+    fn_.call(scope, global_value, &[]).unwrap();
```

Comment:
Why go through all the trouble to pass the global object as the `this` value when `pollForMessages` doesn't use `this`?

Reply:
You're right